task-management/src/main/webapp/skin/css/tasksCommentDrawer.less

```diff
@@ -286,7 +286,7 @@
     border-right: 5px solid @primaryColor ~'!important; /** orientation=rt */ ';
     padding: 10px;
 
-    p {
+    p, ul li, ol li {
```

Comment:
I 'm not sure that the ol & li should use italic font style.
Is this the same style used in CKEditor ?
BTW, you can add more CSS files in iframe/div of CKEditor.

Reply:
> Is this the same style used in CKEditor ?
Yes, it is applied to all quote content https://github.com/Meeds-io/commons/blob/develop/commons-extension-webapp/src/main/webapp/ckeditorCustom/contents.css#L37